sound/soc/intel/boards/bxt_pcm512x.c

```diff
@@ -190,6 +190,12 @@ static const struct snd_soc_ops aif1_ops = {
 	.shutdown = aif1_shutdown,
 };
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_VIRTIO_VHOST)
```

Comment:
We will have a generic virtual VFE machine driver. It should be simple and take all DAI links, PCMs etc from topology.

Reply:
@lgirdwood This is for vhost, i.e. the host side

Reply:
Still neds to use existing machine driver unmodified.

Reply:
that's true, this is a part of the hard-coded stringified configuration part